debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/UnchangedToastedArrayPlaceholder.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class UnchangedToastedArrayPlaceholder {
```

Comment:
Would it be possible to rename this class to `UnchangedToastedPlaceholder` and move `String` and `byte[]` handling there so it is all in the same place?